compiler/rustc_borrowck/src/diagnostics/region_errors.rs

```diff
@@ -694,4 +698,24 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             );
         }
     }
+
+    fn suggest_adding_lifetime_params(
+        &self,
+        diag: &mut Diagnostic,
+        sub: RegionVid,
+        sup: RegionVid,
+    ) {
+        if let (Some(sub), Some(sup)) = (self.to_error_region(sub), self.to_error_region(sup)) {
```

Comment:
Nit: this function can use let else to reduce nesting and clean it up

Reply:
Fixed